lib/adapters/__init__.py

```diff
@@ -0,0 +1,5 @@
+from binance import BinanceAdapter
```

Comment:
This module name will conflict with the actual `binance` package providing the underlying API wrapper.